### PR TITLE
AO3-5673 fandoms | users | works numbers on homepage are hard to parse

### PR DESCRIPTION
**Pull request by dlhardin**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5673 

## Purpose

Adds delimiter (,) to numbers, grouped by thousands.

## Testing Instructions

Display only, no functionality change. Standard library method number_with_delimiter.

## Credit

dlhardin, she/her

### CODE REVIEW

**Comment by dlhardin** [comment]
Sorry about that, still learning. Should be fixed now to have only this commit.

**Comment by sarken** [comment]
It's all good -- extra commits happen to everyone, no matter how experienced!

**Comment by redsummernight** [comment]
Hey @dlhardin, if you have an account on our Jira, please let us know and we'll make sure the issue is assigned to you and your account has the proper permissions.
